pysatModels/utils/match.py

```diff
@@ -166,6 +166,11 @@ def collect_inst_model_pairs(start, stop, tinc, inst, inst_download_kwargs={},
     matched_inst = None
 
     # Test the input
+    if 'model_inst' not in model_load_kwargs.keys():
+        raise ValueError('must provide a pysat.Instrument object')
+    elif not hasattr(model_load_kwargs['model_inst'], 'load'):
+        raise ValueError('must provide a pysat.Instrument object')
+
```

Comment:
Ah, no this isn't how we want to test things in this routine.  Any keyword arguments for the routine may be routine specific.  These don't need to be included.  This routine works equally well with model_load_kwargs={'filename': 'name_of_file'}